.teamcity/Gradle_Check_dsl.iml

```diff
@@ -1,102 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I remember keeping the `.iml` file intentionally under version control. Is that not necessary any more? Shall we add this file to `.gitignore` if it isn't already added?

Reply:
There might be something wrong with my setup. I removed this accidental change.